public/scripts/raneto.js

```diff
@@ -18,8 +18,7 @@
       $(".content table").addClass("table");
 
       // FitVids
-      $(".content").fitVids();
-
+      fitVids();
```

Comment:
I'm glad you caught this because the behavior does change slightly between the two. 
By default it wraps all videos on the page. To keep the behavior the same use `fitvids(".content");`

Want me to update it?